packages/flutter/lib/src/painting/_network_image_io.dart

```diff
@@ -122,3 +139,63 @@ class NetworkImage extends image_provider.ImageProvider<image_provider.NetworkIm
   @override
   String toString() => '$runtimeType("$url", scale: $scale)';
 }
+
+class _DownloadRequest {
+  _DownloadRequest(this.sendPort, this.uri, this.headers,
```

Comment:
No need to wrap at 80.  Either put this all on one line, or put each param on its own line (the way dartfmt would with a trailing comma in the arguments).

Reply:
Done